Projectile mod element and Remove ranged item mod element (#3456)
MCreator/MCreator: plugins/mcreator-localization/help/default/item/enable_ranged_item.md

> By enabling this parameter, the item will become a ranged item allowing it to shoot a projectile, like bows and crossbows.

Reviewer: Установите этот флажок, чтобы предмет можно было использовать для дальнего боя подобно луку, арбалету, снежку или огненному шару.
```suggestion
If you check this box, the item will become a ranged item allowing it to shoot a projectile, like bows and crossbows.
```
